airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -202,9 +205,13 @@ protected void maybeSetShorterConnectionTimeout(final JsonNode config) {
 
   @AfterEach
   public void tearDown() {
+    customCleanUp();
```

Comment:
Why the need for customCleanup and dropSchemas here ? We should at least group them into one.
Looks like the customCleanup code is only used to delete tables, schemas, databases. Please tell me if I'm missing something.
But if I'm right, one of the optimizations that @postamar did for certified connectors was to not delete objects at the end of tests. Since the container is destroyed, we don't care at all. Any reason we can't use the same logic, and get rid of customCleanup.
dropSchemas is already unused, so I think getting rid of it makes more sense than having 2 hooks here.
Alternatively, if we _really_ need to clean things up, can't we use testdb.close() to do that?

Reply:
That's not a correct assumption, there are 3 patterns right now :
1. A new container for each test
2. Same container being re-used across all the tests so they would be required to be clean up
3. Tests for connectors like snowflake, redshift, etc. which dont have a concept of containers which again would require cleanup

Reply:
1) is solved by not doing anything
2) is solved by having a different DB for each test (it's much cheaper to create a DB than to do a cascade delete)
3) would be solved by implementing the DROP CASCADE in the `SnowflakeTestDatabas.close()` method.